presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1587,6 +1587,33 @@ public void testRows()
                 "SELECT 1, cast(null as bigint)");
     }
 
+    @Test
+    public void testRowWithReservedKeywords() throws Exception
+    {
+        // Test: Simple cast with quoted reserved keyword as field name.
+        assertQuery("SELECT col1.\"select\" FROM (SELECT CAST(ROW(11) as ROW(\"select\" INTEGER)) as col1)",
```

Comment:
Some of these are not specific to Hive or do not require connector-specific functionality. They should go in the engine tests (see the tests in com.facebook.presto.sql.query).

Reply:
Will make the change.